sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/RegexpExpressionsSuite.scala

```diff
@@ -498,15 +511,6 @@ class RegexpExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
       StringSplit(Literal("\"quote"), Literal("\"quote"), Literal(-1)) :: Nil)
   }
 
-  test("SPARK-30759: cache initialization for literal patterns") {
```

Comment:
why need remove this suite?

Reply:
It has not been removed and is listed separately. It obtained the pattern in Java regular, but the joni regular engine pattern is private and cannot be obtained.

Reply:
![image](https://user-images.githubusercontent.com/46274164/232367159-bed99026-5335-4953-bf06-547a41ac1406.png)